### ISSUE TITLE
Connection without password is not supported

### ISSUE THREAD

**Issue by dphewa** [dbeaver/dbeaver#6269]
You can't create a connection without a password. If you leave the password field empty, you'll get the following error message:
`Invalid property value '<null>' for property 'password'`

**Comment by uslss** [comment]
Hi @dphewa ,
could you add some additional details like db type?

**Comment by dphewa** [comment]
I'm using a special database (Transbase) which uses a own JDBC driver. I've added the driver via the Driver Manager. My assumption was, that this behavior applies for other db types too.

**Comment by dphewa** [comment]
I did some more investigation. The error `Invalid property value '<null>' for property 'password'` is coming from the JDBC driver. So the driver doesn't accept a null value, but it will accept an empty string.
Is there a way to set the password as empty string?

**Comment by serge-rider** [comment]
Interesting.
Currently we do not support empty password (if no password was specified then dbeaver just skips it). Many databases treats missing password as a special case (no password specified).
I think we need special driver property like "allows empty password". That's easy to implement.

**Comment by serge-rider** [comment]
![image](https://user-images.githubusercontent.com/6398845/62793334-a4f59580-bad1-11e9-8ac7-e67eb60c0f98.png)

**Comment by serge-rider** [comment]
Please test it in Early Access version (https://dbeaver.io/files/ea)

**Comment by dphewa** [comment]
Working as expected. Thank you 👍

**Comment by gitressa** [comment]
Enabling "Allow empty password" under "Driver Settings" don't work for me, but changing the PostgreSQL user's password with this method worked well:

1. log in into the psql console:
    ```
    sudo -u postgres psql
    ```
1. Then in the psql console, change the password and quit:
    ```
    postgres=# \password postgres
    Enter new password: <new-password>
    postgres=# \q
    ```

From [How can I change a PostgreSQL user password?](https://stackoverflow.com/questions/12720967/how-can-i-change-a-postgresql-user-password/12721020#12721020)

**Comment by dkrieger** [comment]
it appears this feature is broken per my experience and that of @gitressa . @uslss could you reopen this issue?

**Comment by ihass** [comment]
> it appears this feature is broken per my experience and that of @gitressa . @uslss could you reopen this issue?

That's what I believed too. In my case I have a PostgreSql running locally on a docker container but only reachable from the container not from my host (Windows laptop) or DBeaver. Thanks to [https://github.com/sameersbn/docker-postgresql/issues/112](url) , I realized that I had a random PostgreSql running on my laptop. I uninstalled it (and I uninstalled ODBC too then deleted "C:\Program Files\PostgreSql\" folder) and now my dockerized PostgreSql is reachable from DBeaver.
